developer-docs-site/docs/sdks/ts-sdk-v2/move-types.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Move Types"
+---
+
+When developing on Aptos, and specifically working with the SDK, developers often need to handle Move types serialization and deserialization. Whether is to construct a transaction payload, build a raw transaction or read BCS data.
+
+The SDK provides a convenient Move sub-classes to easily interact with move types to perform serialization or deserialization operations.
+Each class has a `serialize`, `serializeForEntryFunction` and `serializeForScriptFunction` methods and a `deserialize` static class.
+
+In addition, for complex types like `Vector` the SDK supports nested serialization and deserialization.
+
+### Move primitive types
+
+Classes to handle Move primitive types:
```

Comment:
what about address?

Reply:
we use `AccountAddress` class for that

Reply:
address is actually a move primitive, so its exclusion here is awkward (to me)

Reply:
gotcha. I'll simply add `AccountAddress` to this list and leave a todo comment to use a address move primitive class to extend from